.github/workflows/auto-label.yml

```diff
@@ -0,0 +1,35 @@
+name: Auto Label New Issues and PRs
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
+
+jobs:
+  add-label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add default label to new issues or pull requests
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            // Check if the event is an issue or a pull request
+            if (context.eventName === 'issues') {
+              // For issues, add the "triage" label
+              await github.issues.addLabels({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.payload.issue.number,
+                labels: ['triage']
+              });
+            } else if (context.eventName === 'pull_request') {
+              // For pull requests, add the "ready to review" label
+              await github.issues.addLabels({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.payload.pull_request.number,
```

Comment:
Github already has a mechanism for this... draft pr vs pr 

we just need to doc this new process.. don't move it from draft until you are ready for a review